### PR TITLE
Add TM1637 library

### PR DESCRIPTION
**Pull request by dbrgn**
Is this the same one that you used?

If yes, we should first ask for the license. The repo doesn't contain any.

### CODE REVIEW

**Comment by rnestler** [comment]
> Is this the same one that you used?

I tested it and it worked, so probably the one we used.

**Comment by rnestler** [comment]
> If yes, we should first ask for the license. The repo doesn't contain any.

The author doesn't care about licenses, which is good enough for me: https://github.com/moeur/Arduino_ESP8266/issues/3

File: TM1637.cpp [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    clear();
+}
+
+void tm1637::writeArray()
+{
+    i2cStart();
+    i2cWrite(0xc0);
+    i2cAck();
+	i2cWrite(data[0]);
+	i2cAck();
```
Comment (rnestler) [inline TM1637.cpp]: You may llvm-format the library.

Comment (dbrgn) [reply TM1637.cpp]: Done.

**Review by rnestler — APPROVED** [review]
LGTM. We can still rewrite it using the Arduino TwoWire interface, which may be a bit cleaner.

**Pull request merged**